contents.json

```diff
@@ -17200,6 +17200,25 @@
       ],
       "date_added": "Dec 10 2018",
       "suggested_by": "@marty-suzuki"
-     }
+     },
+     {
+      "title": "AppleWatchFaces",
+      "category-ids": [
+        "apple-watch"
+      ],
+      "description": "Apple Watch Face Editor iOS and watchOS apps",
```

Comment:
Hi @orff, could you use another description text? This one doesn't explain what the app does.

Reply:
Will do